docs/articles/documentation/getting-started/index.md

```diff
@@ -0,0 +1,195 @@
+---
+layout: docs
+title: Getting Started
+permalink: /documentation/getting-started/
+---
+# Getting Started
+
+This guide will provide you with step-by-step instructions on how to create a functional web test with TestCafe.
+You will learn how to install TestCafe, write a test, run the test on multiple browsers and view test results.
+During creating a test, you will learn the basics of TestCafe API, advancing step by step, from the simple to more elaborate testing techniques.
```

Comment:
Do we really need this? Is there someone who don't know what "Getting started" is?